File: FIL/FIL.Web.Admin/ViewModels/CreateEventV1/EventHostsViewModel.cs
using System;
using System.Collections.Generic;

namespace FIL.Web.Kitms.Feel.ViewModels.CreateEventV1
{
  public class EventHostsViewModel
  {
    public bool Success { get; set; }
    public long EventId { get; set; }
    public bool IsValidLink { get; set; }
    public bool IsDraft { get; set; }
    public string CompletedStep { get; set; }
    public short CurrentStep { get; set; }
    public List<FIL.Contracts.DataModels.EventHostMapping> EventHostMapping { get; set; }
  }
}

File: FIL/FIL.Web.Admin/ClientApp/models/CreateEventV1/EventHostsViewModel.ts

export class Host {
  eventId: number;
  id: number;
  altId: string;
  firstName: string;
  lastName: string;
  email: string;
  description: string;
  isEnabled: boolean;
  importantInformation?: string;
}

export class EventHostsViewModel {
  eventHostMapping: Host[];
  success: boolean;
  isValidLink: boolean;
  isDraft: boolean;
  eventId: number;
  currentStep?: number;
  completedStep?: string;
}
